app/models/room_guest.rb

```diff
@@ -0,0 +1,4 @@
+class RoomGuest < ApplicationRecord
+  belongs_to :guest
+  belongs_to :room
```

Comment:
You got the M2M set up! Nice work. I know it was a challenge but I'm very confident you'll be able to get this down pat with a bit more practice!